conf/flash_modes.xml

```diff
@@ -47,6 +47,7 @@
       <board name="li[s]?a_mx_.*"/>
       <board name="px4fmu_.*"/>
       <board name="elle*"/>
+      <board name="lia_.*"/>
```

Comment:
lia board should be handled by the general lisa regexp above. If it is not the case, better fix it.